src/SFA.DAS.AAN.Hub.Data/Entities/Member.cs

```diff
@@ -3,6 +3,19 @@
 public class Member
 {
     public Guid Id { get; set; }
-    public string UserType { get; set; } = null!;
+    public UserType UserType { get; set; }
     public string Email { get; set; } = null!;
+    public string Status { get; set; } = null!;
```

Comment:
We should use Enum here so else where in logic we dont have to use `ToString`

Reply:
Prop changed to enum as requested